src/products/src/products-service/routes.go

```diff
@@ -53,4 +53,16 @@ var routes = Routes{
 		"/products/category/{categoryName}",
 		CategoryShow,
 	},
+	Route{
+		"ProductUpdate",
+		"POST",
+		"/products/",
+		UpdateProduct,
+	},
+	Route{
+		"InventoryUpdate",
+		"POST",
```

Comment:
Should this be a PUT rather than POST since we are updating an existing product?

Reply:
corrected to PUT